templates/user/profile.tmpl

```diff
@@ -69,6 +69,17 @@
 								</ul>
 							</li>
 							{{end}}
+							{{if .Badges}}
+							<li>
+								<ul class="user-badges">
+								{{range .Badges}}
+									<li>
+										<img src="{{.ImageURL}}" alt="{{.Description}}" />
```

Comment:
Probably better to use also have a `tooltip` for description, `alt` attribute is generally invisible.

Reply:
And also should the size of image be fixed?